public/api/conf/1.0/schemas/subSchemas/allocatedReactivations.json

```diff
@@ -1,17 +1,13 @@
 {
   "type": "object",
   "properties": {
-    "ap1NetDisallowances": {
```

Comment:
Missed a "ap1NetDisallowances" in the RAML. public/api/conf/1.0/examples/return/full/request.json